react-graphql-client-library.md

```diff
@@ -23,31 +23,31 @@ summary = "A tutorial on how to build your own GraphQL client for React applicat
 
 {{% read_before_2 "This tutorial is part 3 of 3 in this series." "Part 1:" "A complete React with GraphQL Tutorial" "https://www.robinwieruch.de/react-with-graphql-tutorial/" "Part 2:" "A complete React with Apollo and GraphQL Tutorial" "https://www.robinwieruch.de/react-graphql-apollo-tutorial/" %}}
 
-In your previous applications, you may have used a GraphQL client library, which is potentially partly view layer agnostic, but can be used with React or other view layer solutions. There are a couple of those GraphQL client libraries out there: Apollo Client, Urql and Relay. In the following, I want to show you that it isn't too difficult to implement your own GraphQL client library that works with React. However, the takeaway shouldn't be "build your own GraphQL client for your production ready applications" but rather:
+You may have used a GraphQL client library that was at least partly view layer agnostic, but still able to work with React or other view layer solution in the form of applications like Apollo Client, Urql and Relay. The next chapter is to illustrate how to implement a GraphQL client library that works with React. The takeaway shouldn't be "build your own GraphQL client for your production ready applications", however, you should instead learn:
 
-* see how a GraphQL client library works under the hood
-* see how it isn't too difficult to implement it your way
-* see how you would be able to contribute to the GraphQL ecosystem
+* how a GraphQL client library works under the hood
+* how simple a GraphQL client library is to implement your way
+* how it is capable of contributing to the GraphQL ecosystem
```

Comment:
how it is capable of contributing to the overarching GraphQL ecosystem

Reply:
The sentence I put is the one you asked me in the original notes, which I thought was fine.  This correction makes the sentence sound worse. "Overarching" is a vague word that's unnecessary here because the word "ecosystem", which lets the reader know the scale of what's being discussed.  The latter part of the sentence is also unnecessary--you don't need to say "by publishing it as a contribution to it" if you have the word "contributing" earlier in the sentence, because they mean the same thing in this context.  Software developers who read this book will know what you mean without it.

Reply:
Accepted!